brozzler/chrome.py

```diff
@@ -177,6 +177,9 @@ def start(self, proxy=None, cookie_db=None, disk_cache_dir=None,
                 '--disable-web-security', '--disable-notifications',
                 '--disable-extensions', '--disable-save-password-bubble']
 
+        extra_chrome_args = os.environ.get('BROZZLER_EXTRA_CHROME_ARGS')
+        if extra_chrome_args:
+            chrome_args.append(extra_chrome_args)
```

Comment:
Hmm, I think you're gonna need to split that on white space in case `BROZZLER_EXTRA_CHROME_ARGS='--arg1 --arg2'`

Reply:
I've checked this specific case and its no problem.
If you use `--arg1 --arg2` it will be added to `chrome_args` like all the rest of the parameters and the final chrome command will have `--arg1 --arg2`.

Initially, I considered doing:
```
if extra_chrome_args:
    chrome.args.extend(extra_chrome_args.split(' '))
``
but its exactly the same result and the one we'll have now and the current code is simpler.

Reply:
You need the split. Also you should just do `extra_crome_args.split()` to handle any kind of whitespace.

Compare:
```
>>> import subprocess
>>> subprocess.Popen(['ls', '-a', '-l'])
<subprocess.Popen object at 0x10dcc03d0>
>>> total 0
drwxr-xr-x   4 nlevitt  wheel  128 Nov 27 10:34 .
drwxrwxrwt  15 root     wheel  480 Nov 27 10:34 ..
-rw-r--r--   1 nlevitt  wheel    0 Nov 27 10:34 .a
-rw-r--r--   1 nlevitt  wheel    0 Nov 27 10:34 b
```
```
>>> subprocess.Popen(['ls', '-a -l'])
<subprocess.Popen object at 0x10dcc08d0>
>>> ls: illegal option --  
usage: ls [-ABCFGHLOPRSTUWabcdefghiklmnopqrstuwx1] [file ...]
```